### PR TITLE
added check for manifest in exporting/saving, error message

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: src/core/popcorn-wrapper.js [hunk 1]
```diff
@@ -297,7 +297,12 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
         if ( trackEvents ) {
           for ( i=0, l=trackEvents.length; i<l; ++i ) {
             trackEvent = trackEvents[ i ];
-            popcornOptions = trackEvent._natives.manifest.options;
+            if( trackEvent._natives.manifest ) {
+              popcornOptions = trackEvent._natives.manifest.options;
+            else {
+              popcornOptions = {}
```
Comment (secretrobotron) [inline src/core/popcorn-wrapper.js]: Missing a semicolon.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/aca3cf9fb667c12/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/aca3cf9fb667c12/output.txt

Total script time: 0.60 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @k88hudson received. Current queue size: 0

Live output at: http://butyr.org:8000/e0ca319463cc948/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/e0ca319463cc948/output.txt

Total script time: 0.45 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/f9b7cc2c64c9fea/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/f9b7cc2c64c9fea/output.txt

Total script time: 0.86 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/34c39623a9a7ade/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/34c39623a9a7ade/output.txt

Total script time: 0.47 mins

Server is running on http://pr291.butyr.org/templates/test.html

File: src/core/popcorn-wrapper.js [hunk 2]
```diff
@@ -297,7 +297,12 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
         if ( trackEvents ) {
           for ( i=0, l=trackEvents.length; i<l; ++i ) {
             trackEvent = trackEvents[ i ];
-            popcornOptions = trackEvent._natives.manifest.options;
+            if( trackEvent._natives.manifest ) {
+              popcornOptions = trackEvent._natives.manifest.options;
+            } else {
+              popcornOptions = {};
+              console.log( "WARNING: There was no manifest for trackEvent:", trackEvent );
```
Comment (cadecairos) [inline src/core/popcorn-wrapper.js]: This could be problematic if there's no console defined... is there another way of exposing this error?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/0c5cb6d41b0e6f5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/0c5cb6d41b0e6f5/output.txt

Total script time: 1.31 mins

**Pull request merged**